Marlin/src/inc/Warnings.cpp

```diff
@@ -455,3 +455,12 @@
     #warning "Auto-assigned K_DIAG_PIN to E7_DIAG_PIN."
   #endif
 #endif
+
+/**
+ * Linear Advance 1.5 - Warn for LIN_ADVANCE + low DEFAULT_EJERK
+ */
+#if ENABLED(LIN_ADVANCE) && defined(DEFAULT_EJERK)
+  #if !HAS_JUNCTION_DEVIATION && DEFAULT_EJERK < 10
```

Comment:
The preprocessor can't interpret decimal numbers like `DEFAULT_EJERK`. And, there is no warning equivalent to `static_assert`. So, this whole PR is simply not possible.

Reply:
Can we revert the changes or clarify the error to alert users to comment out the line then?